packages/app/background-jobs-common/vitest.config.mts

```diff
@@ -5,7 +5,6 @@ import defineConfig from '@lokalise/package-vite-config/package'
 // @ts-ignore
 import packageJson from './package.json'
 
-/* eslint-disable import/no-default-export */
```

Comment:
we don't need a biome override here?

Reply:
Not ask me why, but is not needed on `vitest.config` override, maybe it is disabled by default on biome